kubernetes/cray-istio-operator/values.yaml

```diff
@@ -4,5 +4,5 @@ istio-operator:
 
 kubectl:
   image:
-    repository: dtr.dev.cray.com/loftsman/loftsman
+    repository: arti.dev.cray.com/csm-docker-stable-local/loftsman
```

Comment:
The appropriate image to use for `kubectl` is `artifactory.algol60.net/csm-docker/stable/docker-kubectl:1.19.15`.